client/src/common/styledComponents/List.js

```diff
@@ -0,0 +1,29 @@
+import styled from 'vue-styled-components';
+
+// export const List = styled.div`
+export const List = styled.div`
+  display: flex;
+  flex-wrap: wrap;
+`;
+
+export const ListItem = styled.div`
+  flex: 1 0 calc(21%);
+  border: 1px solid #979797;
+  min-height: 200px;
+  margin: 0 10px 10px 0;
```

Comment:
Can we use multiply of 8 or mult of 10 for small spacing (margins and paddings)?